[PATCH] vmcore-dmesg: don't enforce largefile handling

Breaks build on !largefile toolchains.

Instead simply use whatever mode is selected by toolchain / CFLAGS.

Signed-off-by: Peter Korsgaard <jacmet@sunsite.dk>
---
 vmcore-dmesg/vmcore-dmesg.c |    2 --
 1 file changed, 2 deletions(-)

Index: kexec-tools-2.0.3/vmcore-dmesg/vmcore-dmesg.c
===================================================================
--- kexec-tools-2.0.3.orig/vmcore-dmesg/vmcore-dmesg.c
+++ kexec-tools-2.0.3/vmcore-dmesg/vmcore-dmesg.c
@@ -1,6 +1,4 @@
 #define _XOPEN_SOURCE 600
-#define _LARGEFILE_SOURCE 1
-#define _FILE_OFFSET_BITS 64
 #include <endian.h>
 #include <byteswap.h>
 #include <stdio.h>
